images/binderhub/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:17.04
+FROM ubuntu:17.10
```

Comment:
This hasn't been released yet, I think. Is there a reason you wanted to switch already?

Reply:
Oops. Losing track of what month it is. I switched it for Python 3.6. I'll try something else.

Reply:
17.04 has Python 3.6! You just have to install the python3.6 package instead of the python3 package

Reply:
oh, weird.